$MirOS: ports/mailnews/pine/patches/patch-imap_docs_RELNOTES,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- imap/docs/RELNOTES.orig	Thu Apr 21 00:57:42 2005
+++ imap/docs/RELNOTES	Wed May  4 23:28:07 2005
@@ -45,6 +45,11 @@ mapped to Unicode yen sign.
 There are no user-visible functional enhancements in this version.
 
 
+This version of c-client and uw-imapd has been localized
+for $MirOS: ports/mailnews/pine/patches/patch-imap_docs_RELNOTES,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
+and must not be confused with the original code.
+__________________________________________________________________
+
 Updated: 8 July 2004
 
 imap-2004a is a maintenance release, consisting primarily of critical
